source/main.lua

```diff
@@ -24,12 +24,15 @@ local groundSprite <const> = gfx.sprite.new(groundImage)
 
 -- create ground obstacles player jumps over
 local groundObstacleImage <const> = gfx.image.new(20, 30, gfx.kColorBlack)
-local groundObstacleCount <const> = 5 -- amount of obstacles in each "round"
-local baseGroundObstacleSpeed <const> = -2 -- obstacles are moving left, so this is negative
+local groundObstacleImage2 <const> = gfx.image.new(20, 80, gfx.kColorBlack)
```

Comment:
i think this should be named more descriptively -- something like `groundObstacleTall` or `tallGroundObstacle` would suffice